token/cli/src/main.rs

```diff
@@ -2682,26 +2723,29 @@ fn main() -> Result<(), Error> {
     };
 
     solana_logger::setup_with_default("solana=info");
-    let result = process_command(&sub_command, matches, &config, wallet_manager, bulk_signers)
-        .map_err::<Error, _>(|err| DisplayError::new_as_boxed(err).into())?;
```

Comment:
`DisplayError` is an issue later since it doesn't have `Send + Sync`. What's it about exactly? In this case, since we already have a `Box<dyn Error>`, it seems redundant to just wrap it in a `DisplayError`, but I'm probably missing something.

Reply:
@t-nelson can probably give more color here.
However, can we not await before mapping the error?

Reply:
oh god... so long ago.  iirc `DisplayError` was introduced as a shim to prevent exposing CLI users to `Debug` output on error, which is how any `Result<T,E>::Err(E)` is presented.  Instead it implements `Debug` to output the `Display` string